src/mgr/DaemonServer.cc

```diff
@@ -2320,9 +2320,12 @@ void DaemonServer::_send_configure(ConnectionRef c)
 {
   assert(lock.is_locked_by_me());
 
+  OSDPerfMetricQuery query;
+
   auto configure = new MMgrConfigure();
   configure->stats_period = g_conf().get_val<int64_t>("mgr_stats_period");
   configure->stats_threshold = g_conf().get_val<int64_t>("mgr_stats_threshold");
+  configure->query_list.push_back(query); 
```

Comment:
I suppose we want add it only when `c->peer_is_osd()` is true. But this can be fixed later too.